_posts/2024-07-22-kubeflow-1.9-release.md

```diff
@@ -0,0 +1,230 @@
+- **vLLM Support**: Dedicated runtime support for [vLLM](https://docs.vllm.ai/en/latest/) is now included, streamlining the deployment process for LLMs.
+- **OpenAI Schema Integration**: KServe now supports endpoints for generative transformer models, following the OpenAI protocol.  This enables KServe to be used directly with OpenAI’s client libraries or third-party tools like LangChain and LlamaIndex.
+
+## Documentation
+
+MLOps is a complex subject and users have asked for clear, up-to-date and comprehensive documentation. We are happy to announce that we started a restructuring process to better align the various components’ docs to have a similar structure. We are revamping our docs to better align with user expectations and how you expect technical docs to be organized. We will continue to improve the quality and completeness of our docs, by adding new user guides, tutorials, and reference architecture topics.
+
+We are looking for new members who can help us craft complete and high quality documentation. Please get involved by [opening and reviewing PRs](https://github.com/kubeflow/website/pulls) in the Kubeflow website.
+
+## Honorable Mentions
+
+### Google Spark Operator migration to Kubeflow
+
+We’re excited to announce the migration of Google’s Spark Operator to the Kubeflow Spark Operator, marking the launch of a significant addition to the Kubeflow ecosystem. The Kubeflow Spark Operator simplifies the deployment and management of Apache Spark applications on Kubernetes. This announcement isn’t just about a new piece of technology, it’s about building a stronger, open-governed, and more collaborative community around Spark on Kubernetes.
+
+Kubeflow Spark Operator is not yet officially included in the Kubeflow release, but you can install it by following the instructions [here](https://www.kubeflow.org/docs/components/spark-operator/getting-started/).
+
+Read more about Kubeflow Spark operator in the announcement [blog post](https://blog.kubeflow.org/operators/2024/04/15/kubeflow-spark-operator.html).
+
+### Google Summer of Code
+
+This year, Kubeflow was excited to participate in Google Summer of Code (GSoC), attracting a wave of enthusiastic students! Over 250 students joined our Slack channel, eager to learn about contributing to the Kubeflow community and crafting impactful proposals.
+
+We were also fortunate to have a dedicated group of 20 mentors ready to guide these talented individuals. From a pool of nearly 70 proposals, we have selected 10 proposals and were awarded with 9 outstanding students by Google. They are now actively contributing to various Kubeflow features, making a real difference in various Kubeflow components.
+
```

Comment:
@StefanoFioravanzo @rareddy should we consider listing the GSOC students by name ?    It might help their careers and keep them engaged in the community.

Reply:
Perhaps include a link to the Kubeflow GSoC from the website which has all the info for students, mentors, and projects.